doc/languages-frameworks/texlive.xml

```diff
@@ -59,6 +59,27 @@ nix-repl> texlive.collection-<TAB>
   </itemizedlist>
  </section>
 
+ <section xml:id="sec-language-texlive-custom-packages">
+  <title>Custom packages</title>
+  <para>
+    You may find that you need to add your own package to the combination. This is currently not handled in a user-friendly way by the combine function. Here is a hackish way to do it, as an example:
+<programlisting>
+let
+  customDerivation = runCommand "customDerivation" {
```

Comment:
I suppose a typical derivation in nixpkgs would use `stdenvNoCC.mkDerivation` to unpack downloaded archive and copy files to the output.

Reply:
Changed